data_sources/PreProd Youth Permissions.csv

```diff
@@ -1,165 +1,166 @@
+02382,Brockton,brockton-admin1@example.com,brockton-admin2@example.com,brockton-admin3@example.com,brockton-admin4@example.com,,,,,,,notificationemail@example.com
+02445,Brookline,brookline-admin1@example.com,brookline-admin2@example.com,brookline-admin3@example.com,brookline-admin4@example.com,,,,,,,notificationemail@example.com
+02446,Brookline,brookline-admin1@example.com,brookline-admin2@example.com,brookline-admin3@example.com,brookline-admin4@example.com,,,,,,,notificationemail@example.com
+02447,Brookline,brookline-admin1@example.com,brookline-admin2@example.com,brookline-admin3@example.com,brookline-admin4@example.com,,,,,,,notificationemail@example.com
+02138,Cambridge,cambridge-admin1@example.com,cambridge-admin2@example.com,cambridge-admin3@example.com,cambridge-admin4@example.com,,,,,,,notificationemail@example.com
+02139,Cambridge,cambridge-admin1@example.com,cambridge-admin2@example.com,cambridge-admin3@example.com,cambridge-admin4@example.com,,,,,,,notificationemail@example.com
+02140,Cambridge,cambridge-admin1@example.com,cambridge-admin2@example.com,cambridge-admin3@example.com,cambridge-admin4@example.com,,,,,,,notificationemail@example.com
+02141,Cambridge,cambridge-admin1@example.com,cambridge-admin2@example.com,cambridge-admin3@example.com,cambridge-admin4@example.com,,,,,,,notificationemail@example.com
+02142,Cambridge,cambridge-admin1@example.com,cambridge-admin2@example.com,cambridge-admin3@example.com,cambridge-admin4@example.com,,,,,,,notificationemail@example.com
+02238,Cambridge,cambridge-admin1@example.com,cambridge-admin2@example.com,cambridge-admin3@example.com,cambridge-admin4@example.com,,,,,,,notificationemail@example.com
+02150,Chelsea,chelsea-admin1@example.com,chelsea-admin2@example.com,chelsea-admin3@example.com,chelsea-admin4@example.com,,,,,,,notificationemail@example.com
+02149,Everett,everett-admin1@example.com,everett-admin2@example.com,everett-admin3@example.com,everett-admin4@example.com,,,,,,,notificationemail@example.com
+01701,Framingham,framingham-admin1@example.com,framingham-admin2@example.com,framingham-admin3@example.com,framingham-admin4@example.com,,,,,,,notificationemail@example.com
+01702,Framingham,framingham-admin1@example.com,framingham-admin2@example.com,framingham-admin3@example.com,framingham-admin4@example.com,,,,,,,notificationemail@example.com
+02420,Lexington,lexington-admin1@example.com,lexington-admin2@example.com,lexington-admin3@example.com,lexington-admin4@example.com,,,,,,,notificationemail@example.com
+02421,Lexington,lexington-admin1@example.com,lexington-admin2@example.com,lexington-admin3@example.com,lexington-admin4@example.com,,,,,,,notificationemail@example.com
+02148,Malden,maldentestuser@example.com,malden-admin2@example.com,malden-admin3@example.com,malden-admin4@example.com,,,,,,,notificationemail@example.com

```

Comment:
We can change the maldentestuser email to follow the same naming convention. We no longer use the Malden automation account due to problems with two-factor login

Reply:
Great point. First Malden email now fits the naming convention.